cli/src/test/resources/ide-projects/intellij/repository/intellij/intellij/default/mac/bin/idea.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "intellij mac $*"
```

Comment:
The structure of the MacOS IntelliJ IMHO is very different. Also your code that is trying to create a Path pointing to `Contents/MacOS/idea` but that does not exist at all.
Therefore the test is not realistic and could be improved.
I am very aware that we also need a manual test or automated e2e test to really verify that this code is able to install and run IntelliJ on MacOS.